assembl/static2/js/app/components/administration/menu.jsx

```diff
@@ -24,6 +24,13 @@ class Menu extends React.Component {
                 </span>
               </Link>
             </li>
+            <li>
+              <Link to={`${get('administration', slug)}/discussion?section=3`} activeClassName="active">
```

Comment:
Create a new key in `routeMap`?

Reply:
If you add a route, maybe add it after #252 is merged, the routes are now in routes.json

Reply:
@MilanBarande #252 is merged, you can merge develop to this branch and add a route in routes.json